modules/rosa-deleting-cluster.adoc

```diff
@@ -32,6 +46,34 @@ $ rosa init --delete-stack
 ----
 endif::sts[]
 
+ifdef::sts[]
+.  Delete the OpenID Connect (OIDC) provider that the cluster Operators use to authenticate:
++
+[source,terminal]
+----
+$ rosa delete oidc-provider -c <cluster_id> --mode auto --yes <1>
+----
+<1> Replace `<cluster_id>` with the ID of the cluster.
+
+. Delete the cluster-specific Operator IAM roles:
++
+[source,terminal]
+----
+$ rosa delete operator-roles -c <cluster_id> --mode auto --yes <1>
+----
+<1> Replace `<cluster_id>` with the ID of the cluster.
+
+. Delete the account-wide roles:
++
+[source,terminal]
+----
+$ rosa delete account-roles --prefix <prefix> --mode auto --yes <1>
+----
+<1> Replace `<prefix>` with the prefix of the account roles that are being deleted, for example `ManagedOpenShift`.
+
+. Delete the account-wide inline and Operator policies by using the AWS IAM Console. For detailed steps, see the _Deleting the AWS resources by using the AWS IAM Console_ section.
```

Comment:
This entire section should come with a big warning. Deleting account-wide roles/policies implies that they no longer have any STS cluster in their account, nor plan on creating new clusters (otherwise they would have to start over by creating new roles/policies). Should this be under a section different from "deleting cluster"?

cc @arendej

Reply:
@vkareh thank you for your review and for highlighting this. I could repeat [the important admonition that is in the introduction to this section](https://github.com/openshift/openshift-docs/pull/39632/files#diff-249fa49d6253b4c3c9e717a16f5d10d5b02a7f0a4223d314e77514916a88bf66R18-R21) after this step, to make this even clearer perhaps?

Reply:
Yes this might be a good idea to point out closer to the command in question, just to be sure. Deleting account-wide roles is a very destructive action.

Reply:
@vkareh I have duplicated the important admonition in the "Delete the account-wide roles" step directly. I have also kept it in place in the introduction to this section, so that the reader has an opportunity to see it before starting the procedure and in context to that step directly too.

For the step after that, on deleting the account-wide inline and Operator policies, the section that is referenced also includes the same important admonitions. I thought it best to include it under only one of these steps in this deletion procedure instead of duplicating it twice, given that the customer will see the admonition in the referenced section too.